glide.yaml

```diff
@@ -6,14 +6,13 @@ owners:
   email: rob@tigera.io
 import:
 - package: github.com/Sirupsen/logrus
-  version: ^0.10.0
```

Comment:
Unless 1.0.0 has complicated breaking changes, feels like we should just upstream our downstream projects so they pin to ^1.0.0, yeah?